images/twitter.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Twitter</title>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+
+  <body>
+    <div id= "twitterContainer">
+        <div class="left">
+          <div class="logoTwitter">
+            <img src="/asect/twitterlogo.webp" alt="Twitter logo" width="60px">
+          </div>
+          <div class="menuItem">
+         
+            <div class="menuLogo"><img src="/asect/home1.webp" alt="Home" width="30px"> </div>
+             <div class="menuTxt">Home</div>
+
+             <div class="menuLogo"><img src="/asect/explore.svg.png" alt="Explore"  width="30px" > </div>
+             <div class="menuTxt">Explore</div>
+
+             <div class="menuLogo"><img src="/asect/notification-icon.jpg" alt="Home"  width="20px"> </div>
+             <div class="menuTxt">Notifications</div>
+
+             <div class="menuLogo"><img src="/asect/18609.png" alt="Messages" width="20px"> </div>
+             <div class="menuTxt">Messages</div>
+
+             <div class="menuLogo"><img src="/asect/bookmark.png" alt="Bookmarks"  width="10px" > </div>
+             <div class="menuTxt">Bookmarks</div>
+
+             <div class="menuLogo"><img src="/asect/profile.png" alt="Profile"  width="20px" > </div>
+             <div class="menuTxt">Profile</div>
+
+             <div class="menuLogo"><img src="/asect/more.png" alt="More"  width="20px"> </div>
+             <div class="menuTxt">More</div>     
+        
+          </div>
+          <button class="twittButton">Twitte</button>
+          <div class="leftProfilebottom">
+            <div class="leftProfilePic">
+                <img src="/asect/handsome-man-black-suit-white-shirt-posing-studio-attractive-guy-fashion-hairstyle-confident-man-short-beard-125019349.jpg" alt="profile pic" width="50px">
+            </div>
+            <div class="leftprofiletxt"><p class="profileName">John Doe</p><p class="twitterhandle"> @jdoe</p></div>
+           
+          </div>
+        </div>
+        <main class="mainFeedTwitter">
+          <h2>Home</h2>
+          <div class="story">
+            <div class="image">
+              <img
+              src="/asect/handsome-man-black-suit-white-shirt-posing-studio-attractive-guy-fashion-hairstyle-confident-man-short-beard-125019349.jpg"
+              alt="manimg"
+              width="60px"/>
+            </div>
+            <div class="inputfield">What's happening?</div>
+            <div class="image2">
+              <img
+                src="/asect/mainimage.svg.jpg"
+                alt="mail"
+                width="150px"
+                height="30px"
+              />
+              <button class="twitbutton">Twitte</button>
```

Comment:
try using align and justify to position elements where you want them to rather than using margin for it.